test/Sentry.Google.Cloud.Functions.Tests/Sentry.Google.Cloud.Functions.Tests.csproj

```diff
@@ -2,6 +2,8 @@
 
   <PropertyGroup>
     <TargetFrameworks>net5.0;netcoreapp3.1</TargetFrameworks>
+    <!-- Tests are failing on net6.0-->
+    <!-- <TargetFrameworks>net6.0;net5.0;netcoreapp3.1</TargetFrameworks>-->
```

Comment:
@lucas-zimerman tests fail on net6.0. Since you've been working on this, do you mind making a PR into this with a fix?

Reply:
Sure

Reply:
Fix here : https://github.com/getsentry/sentry-dotnet/pull/1294